dash.el

```diff
@@ -2154,6 +2154,23 @@ N is the length of the returned list."
         (push (funcall fun (car r)) r))
       (nreverse r))))
 
+(defun -iota (count &optional start step)
+  "Return a list containing COUNT numbers.
+Starts from START and adds STEP each time.  The default START is
+zero, the default STEP is 1.
+This function takes its name from the corresponding primitive in
+the APL language."
+  (if (or (not (integerp count)) (< count 0))
```

Comment:
AKA `(not (natnump count))`.

Reply:
Pretty sure I wrote it this way because natnup lacks a bytecode

Reply:
So? It's still faster to only call 2 functions rather than 3 plus a special form.